utils/templates/test_module+member.yml.in

```diff
@@ -20,6 +20,7 @@
   - name: Ensure $name NAME is present
```

Comment:
Line 3 is not correct. It should be `hosts: "{{ ipa_test_host | default('ipaserver') }}"`

Reply:
Fixed line 3 (but see below).